main/py-chardet/APKBUILD

```diff
@@ -2,12 +2,12 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 _pkgname=chardet
 pkgname=py-$_pkgname
-pkgver=2.3.0
-pkgrel=3
+pkgver=3.0.3
+pkgrel=0
 pkgdesc="The Universal Character Encoding Detector"
 url="https://github.com/chardet/chardet/"
 arch="noarch"
-license="LGPL-2.1"
+license="LGPL 2.1"
```

Comment:
What's the reason for this? I'm unsure what we currently use as license separator, but I think it's space in a few packages.